__tests__/app.test.js

```diff
@@ -106,3 +105,39 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET  /api/articles/:article_id/comments", () => {
+  test("200: should return array of object comments", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toHaveLength(11);
+        body.comments.forEach((comment) => {
+          expect(comment).toHaveProperty("comment_id", expect.any(Number));
+          expect(comment).toHaveProperty("votes", expect.any(Number));
+          expect(comment).toHaveProperty("created_at", expect.any(String));
+          expect(comment).toHaveProperty("author", expect.any(String));
+          expect(comment).toHaveProperty("body", expect.any(String));
+          expect(comment).toHaveProperty("article_id", expect.any(Number));
```

Comment:
For this line, would you expect the article_id to be any number? Be more specific here as to what article_id you're expecting the comments to have

Reply:
done